### ISSUE TITLE
keycloak in the same Caddyfile

### ISSUE THREAD

**Issue by omario36** [authp/authp.github.io#42]
Hi Team,

Is it possible to have on the same Caddyfile the keycloak service and other services to secure with the cadd-security ? 

the metadata_url  try to check if keycloak is up before running the caddy ...

**Comment by jb-home** [comment]
Yes, for that I am using the internal URL of the keycloak. Not the external URL which caddy would publish.
